docs/source/roles/researcher/troubleshooting.md

```diff
@@ -0,0 +1,280 @@
+(roles_researcher_sre_troubleshooting)=
+
+# Troubleshooting
+
+## {{musical_keyboard}} Keyboard mapping
+
+When you access the workspace you are actually connecting through the cloud to another computer - via a few intermediate computers/servers that monitor and maintain the security of the SRE.
+
+:::{caution}
+You may find that the keyboard mapping on your computer is not the same as the one set for the workspace.
+:::
+
+::::{admonition} Changing the keyboard layout
+:class: dropdown note
+
+- From the workspace desktop, click on **{menuselection}`Applications --> Settings --> Keyboard`** to change the layout.
+
+    :::{tip}
+    We recommend opening an application that allows text entry (such as **Libre Office Writer** or a text editor) to check what keys the remote desktop thinks you're typing – especially if you need to use special characters.
+    :::
+::::
+
+## {{fox_face}} Firefox not responding
+
+If you get an error message like this one about Firefox not responding, it is likely because you have a browser window open on another workspace.
+
+:::{image} images/firefox_not_responding.png
+:alt: Firefox not responding
+:align: center
+:width: 90%
+:::
+
+Either log into that workspace and close Firefox, or follow the instructions [here](https://support.mozilla.org/en-US/kb/firefox-already-running-not-responding#w_remove-the-profile-lock-file) and delete your profile lock file.
+
+:::{tip}
+Your profile is likely stored under **~/snap/firefox/common/.mozilla/firefox**.
+:::
+
+## {{passport_control}} Command Line Basics
```

Comment:
I think it would be better to link to an external resource for this.

Reply:
We do (https://ubuntu.com/tutorials/command-line-for-beginners). Do you want to delete this whole section and just leave this link?

Reply:
Yes I think so. Maybe we want a couple of alternatives to point people to?

Reply:
Done in b772c7a